src/libmongoc/doc/mongoc_uri_t.rst

```diff
@@ -87,20 +87,20 @@ IPv4 and IPv6
-MONGOC_URI_COMPRESSORS                     compressors                       Comma separated list of compressors, if any, to use to compress the wire protocol messages. Snappy, zlib, and zstd are optional build time dependencies, and enable the "snappy", "zlib", and "zstd" values respectively. Defaults to empty (no compressors).
-MONGOC_URI_CONNECTTIMEOUTMS                connecttimeoutms                  This setting applies to new server connections. It is also used as the socket timeout for server discovery and monitoring operations. The default is 10,000 ms (10 seconds).
-MONGOC_URI_SOCKETTIMEOUTMS                 sockettimeoutms                   The time in milliseconds to attempt to send or receive on a socket before the attempt times out. The default is 300,000 (5 minutes).
-MONGOC_URI_REPLICASET                      replicaset                        The name of the Replica Set that the driver should connect to.
-MONGOC_URI_ZLIBCOMPRESSIONLEVEL            zlibcompressionlevel              When the MONGOC_URI_COMPRESSORS includes "zlib" this options configures the zlib compression level, when the zlib compressor is used to compress client data.
-========================================== ================================= ============================================================================================================================================================================================================================================
+========================================== ================================= ================================= ============================================================================================================================================================================================================================================
+Constant                                   Key                               Default                           Description
+========================================== ================================= ================================= ============================================================================================================================================================================================================================================
+MONGOC_URI_RETRYREADS                      retryreads                        true                              If "true" and the server is a MongoDB 3.6+ standalone, replica set, or sharded cluster, the driver safely retries a read that failed due to a network error or replica set failover.
+MONGOC_URI_RETRYWRITES                     retrywrites                       true if driver built w/ SSL       If "true" and the server is a MongoDB 3.6+ replica set or sharded cluster, the driver safely retries a write that failed due to a network error or replica set failover. Only inserts, updates of single documents, or deletes of single
```

Comment:
Use "TLS" instead of "SSL" since the SSL URI options (and server options) are deprecated in favor of the TLS ones.

Reply:
Done.